Code/C++/K_largest_element.cpp

```diff
@@ -0,0 +1,98 @@
+//CPP Code For Finding K Largest Element In An Array In Ascending Order.
+/*
+             Approach
+             ========
+           
+            We can efficiently implement this using a Min-Heap.
+            
+            1. We will create a min heap.
+            2. We will traverse through all the elements of the array 
```

Comment:
Just update here that you are using a min heap and inverting the numbers, i.e. inserting -1*number